tests/test_zeus_client.py

```diff
@@ -68,23 +81,22 @@ def test_validate_dates(self):
     @mock.patch('zeus.interfaces.rest.requests')
     def test_post_empty_log(self, mock_requests):
         logs = []
+        url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)
```

Comment:
```python
posixpath.join('a/', '/b', 'c')
# => '/b/c'

posixpath.join('a/', '/b')
# => '/b'
```

If there are no strong preference, i think `format()` might be good enough? how do you think?